opdracht-2/index.html

```diff
@@ -2,11 +2,155 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <link rel="stylesheet" href="../do-not-edit.css">
-    <link rel="stylesheet" href="styles.css">
+    <link href="../do-not-edit.css" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
     <title>NOVI Flexbox opdracht 2</title>
 </head>
 <body>
-    <p>It's Flexbox time!</p>
+<aside class="nav-bar">
+    <div class="logo">
+        <img alt="Het Novi logo" class="logo-img" src="../assets/logo-acme.svg"/>
+        <p>NOVI Futuristic Inc</p>
+    </div>
+    <nav class="nav-bar-buttons">
+        <ul>
+            <li>
+                <a href="/"><img alt="house" src="../assets/home.svg"/>
+                    <p>Dashboard</p>
+                </a>
+            </li>
+            <li><a href="/"><img alt="app squares" src="../assets/apps.svg"/>
+                <p>Projecten</p></a></li>
+            <li><a class="active" href="/"><img alt="calender" src="../assets/calendar.svg"/>
```

Comment:
```suggestion
            <li><a href="/"><img alt="calender" src="../assets/calendar.svg"/>
```
Ik ga ervan uit dat de class active gebruikt was om te testen. Hierdoor staat wel een van je menu items constant geselecteerd. Dat is ietwat apart.